src/styles/utils/normalize.scss

```diff
@@ -0,0 +1,11 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
Use of align-items on the body tag can affect layout of other elements on the page. Consider applying this to a specific class or id.